codemod-expand-transforms.js

```diff
@@ -0,0 +1,45 @@
+const glob = require('fast-glob')
+const fs = require('fs')
+
+const composeNewLines = line => {
```

Comment:
should we filter out the transforms that views doesn't support? https://github.com/viewstools/morph/blob/master/parse/helpers.js#L95-L102

Reply:
Should we filter out `rotate` as well? If I'm not wrong `rotate` is the same as `rotateZ`...
https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotate